test/studies/shootout/k-nucleotide/benmcd/knucleotide-user-hash.chpl

```diff
@@ -0,0 +1,149 @@
+use IO, Map, Sort;
```

Comment:
I think you should kick this file off with a comment similar to the ones in our other submitted entries:

```
/* The Computer Language Benchmarks Game
   https://salsa.debian.org/benchmarksgame-team/benchmarksgame/

   contributed by Ben McDonald
   derived from the Chapel #3 version by Ben Harshbarger and Brad Chamberlain
*/
```